scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -646,6 +646,20 @@ public boolean restartTask(String jobId, String taskName, int restartDelay)
      */
     @Override
     @ImmediateService
+    public boolean finishInErrorTask(String jobId, String taskName)
+            throws NotConnectedException, UnknownJobException, UnknownTaskException, PermissionException {
+        // checking permissions
+        final JobId jobIdObject = JobIdImpl.makeJobId(jobId);
+        frontendState.checkJobOwner("finishTaskInError", frontendState.getIdentifiedJob(jobIdObject),
```

Comment:
checkowner should be changed to : 

frontendState.checkPermissions("finishTaskInError", frontendState.getIdentifiedJob(jobIdObject),
                YOU_DO_NOT_HAVE_PERMISSION_TO_FINISH_THIS_TASK);